cms/apps/media/views.py

```diff
@@ -0,0 +1,56 @@
+import json
+import magic
+
+from django.contrib.auth.mixins import LoginRequiredMixin
+from django.http import JsonResponse
+from django.template.defaultfilters import filesizeformat
+from django.utils.safestring import mark_safe
+from django.views.generic import View
+
+from cms.apps.media.models import File
+
+
+class ImageUploadView(View, LoginRequiredMixin):
+    def get(self, request):
```

Comment:
With both this and the `post` method I do not see any checks to stop these endpoints from being used directly, i.e. to see whether they have permissions to list and upload files. This is not acceptable if so.

Reply:
(There is LoginRequiredMixin, but that only establishes that they are logged in, and not that they have permissions to upload and list files.)